src/compas/plugins.py

```diff
@@ -33,6 +35,17 @@
     "PluginDefaultNotAvailableError",
 ]
 
+if compas.RHINO:
+    import System  # type: ignore  # pragma: no cover
```

Comment:
i would not add these pragma annotations. we don't really care about the coverage notifications, so i would also not pollute the codebase to reduce their frequency...